### PR TITLE
add some margin between #printing-box and #data-box after moving boxes around

### PR DESCRIPTION
**Pull request by tomltd**
This branch name is too long :deciduous_tree:

### CODE REVIEW

File: src/css/style.css [hunk 1]
```diff
@@ -948,6 +948,10 @@ header span {
   margin-top: 0;
 }
 
+#printing-box {
+  margin-bottom: 1.5em;
+}
+
```
Comment (katjam) [inline src/css/style.css]: I would argue that the container under the printing box should have the default margin-top: 1.5em which is on all body *+* so why doesn't it?

**Comment by katjam** [comment]
Not a big issue - merge if you want. On your own head :hear_no_evil:

**Comment by tomltd** [comment]
I thought that too but when I took the margin: 0 off the data-box it made the sides shink in to fit the content so you have to say margin 0 on left and right or it take it as 20% because of the .box class so I just added the margin to the print-box, but I guess I could have added margin to the top of the data-box to keep things standard. I just tried a bunch of different stuff and it got crazy complicated so might stick with this.

**Comment by katjam** [comment]
smile.

**Comment by tomltd** [comment]
I did it tidy like. Got there in the end :sunglasses:

**Pull request merged**